apps/obsidian/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "@repo/typescript-config/react-library.json",
```

Comment:
needs to be migrated as much as possible to the `@repo/typescript-config`
https://roamresearch.com/#/app/discourse-graphs/page/s6veJ_Zvr